### PR TITLE
fix : #510, Removed package_to_check variable and updated references

### PR DESCRIPTION
**Pull request by shobhit9957**
Fix: #510 
Removed package_to_check variable and updated according to the reference.

### CODE REVIEW

**Comment by mickahell** [comment]
Do not hesitate to ping me when it's ok for you :)

**Comment by shobhit9957** [comment]
Hey, what should I do, how can this can be merged?, I'm a beginner to this open-source community...please help me..

**Comment by mickahell** [comment]
> Hey, what should I do, how can this can be merged?, I'm a beginner to this open-source community...please help me..

For merging we'll do it for you :) just continue to commit and you think your PR is ready just ping me I'll review it and then I'll  ping someone who can merge it.

Here you cleaned the tox tpl, you can as well delete wherever this var is called in the code.

**Comment by shobhit9957** [comment]
So, should I do that too?, like should I delete the var where ever it is in the code?

**Comment by mickahell** [comment]
> So, should I do that too?, like should I delete the var where ever it is in the code?

Yes :)

**Review by mickahell — CHANGES_REQUESTED** [review]
Why did you renamed everywhere the var to `qiskit` ?
You need to delete this var everywhere

**Comment by shobhit9957** [comment]
Oops!, Very Sorry..my bad, I'll do that again..

File: ecosystem/manager.py [hunk 1]
```diff
@@ -324,7 +324,10 @@ def _run_python_tests(
         repo_url: str,
         tier: str,
         python_version: str,
-        package_to_check: str,
+<<<<<<< HEAD
+=======
+        qiskit: str,
+>>>>>>> 2b284cffea465cb413ac35b655efce9f5be03546
```
Comment (Eric-Arellano) [inline ecosystem/manager.py:330]: Hey, it looks like you have a lot of merge conflicts still like this, if you look through the "Files changed" tab on GitHub.

Thanks for your contribution by the way! We appreciate your help.

Comment (shobhit9957) [reply ecosystem/manager.py:330]: Hey, sorry for this..actually i'm a beginner here..learning and applying things. So I made this mistake, so sorry for that...will push the pr again in couple of minutes...Thanks for providing me this opportunity bro.

**Comment by shobhit9957** [comment]
Hey bro.. You can check, if I again made any mistake, please tell me bro.. I'll solve it and push the commit again...

**Review by Eric-Arellano — APPROVED** [review]
This looks good to me! I'll wait for @mickahell to review.

> please tell me bro

Gentle tip: it's better to not use "bro" when interacting with people in developer spaces like GitHub and open source. It's really casual, whereas we try to be a little more professional with GitHub. 

Also not everyone identifies as male, and it's not safe to assume how someone identifies. For example, I'm [non-binary](https://transequality.org/issues/resources/understanding-nonbinary-people-how-to-be-respectful-and-supportive) so "bro" doesn't describe me.

**Comment by shobhit9957** [comment]
Yep!, I'll follow the tip.

**Review by mickahell — APPROVED** [review]
Yeah seems good to me too :)

@Eric-Arellano  could you merge it please 🚀

**Pull request merged**